### ISSUE TITLE
Does "NCBITaxonomy Semantic Type PR" break `classify-kaiju`?

### ISSUE THREAD

**Issue by Sann5** [bokulich-lab/q2-moshpit#117]
TLDR: No. At least I don't think so.

- bokulich-lab/q2-types-genomics#67 moved the location of the definition of two Semantic Types: `NCBITaxonomyNodesFormat` and `NCBITaxonomyNamesFormat`.
- Both `NCBITaxonomyNodesFormat` and `NCBITaxonomyNamesFormat` are file formats specified within `KaijuDBDirectoryFormat` which is then used by the `q2-moshpit` plugin in the `classify-kaiju` action. 
- The only place where `NCBITaxonomyNodesFormat` and `NCBITaxonomyNamesFormat` were called was in `q2_types_genomics/kaiju/_format.py` (where `KaijuDBDirectoryFormat` is defined) but never in `q2-moshpit`, so moving them only required modifications in `q2_types_genomics`, mainly to tests.

**Comment by misialq** [comment]
Thanks for investigating @Sann5! 🙌
